include/lmms_math.h

```diff
@@ -360,6 +360,21 @@ static inline int numDigitsAsInt(float f)
 	return digits;
 }
 
+static inline float minEpsilon()
+{
+	return 1.0e-10f;
+}
+
+static inline bool floatCompare(float x, float y)
+{
+	if(x == y)
+	{
+		return true;
+	}
+	return std::abs(x - y) < minEpsilon();
```

Comment:
You can fix the formatting a bit here (specifically add a space after the `if`) or just go with a one-liner, up to you.
```suggestion
	return x == y ? true : std::abs(x - y) < minEpsilon();
```

Reply:
I often don't think of ternary statements. I'll change.